<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-UG71NEM" ignored="0" name="BankingTest1" tests="5" failures="3" timestamp="2023-08-09T11:57:59 IST" time="149.195" errors="0">
  <testcase name="loginDOt" time="44.804" classname="TestCases.DATLoginTest"/>
  <testcase name="loginDOt" time="21.58" classname="TestCases.DATLoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.DATLoginTest.loginDOt(Unknown Source)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDOt -->
  <testcase name="loginDOt" time="21.397" classname="TestCases.DATLoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.DATLoginTest.loginDOt(Unknown Source)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDOt -->
  <testcase name="loginDOt" time="23.074" classname="TestCases.DATLoginTest"/>
  <testcase name="loginDOt" time="25.652" classname="TestCases.DATLoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.DATLoginTest.loginDOt(Unknown Source)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDOt -->
</testsuite> <!-- BankingTest1 -->
